### PR TITLE
Hw js 4

### PR DESCRIPTION
**Pull request by ZhenyaSin**
Solving tasks "task1" and "task2"

### CODE REVIEW

File: HW_Lection4/Task1.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+//     Если “age”  > age_3, вывести в консоль “Keep calm and look Culture channel”.
+
+//     Иначе выводите “Technical work”.
+
+
+
+let age = 10;
+let age_2 = 18;
+let age_3 = 60;
```
Comment (VladimirLQA) [inline HW_Lection4/Task1.js]: Use `const`

File: HW_Lection4/Task2_1.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// 1.
+// Реализовать Task 1 через switch
+let age = "aaa";
+let age_2 = 18;
+let age_3 = 61;
```
Comment (VladimirLQA) [inline HW_Lection4/Task2_1.js]: Use `const`

File: HW_Lection4/Task2_2.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// 2. 
+// Преобразовать написанный код в task 1 так, чтобы сначала проверялся тип данных. 
+//   И если он не number - кидалась ошибка в консоль.
+//   Проверить работу кода на следующих данных 17, 18, 61, "2", "aaa"
+
+let age = '10';
+let age_2 = 18;
+let age_3 = 60;
```
Comment (VladimirLQA) [inline HW_Lection4/Task2_2.js]: Use `const`

File: HW_Lection4/Task2_2.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+if (typeof age !== 'number' && !isNaN(age)) {
```
Comment (VladimirLQA) [inline HW_Lection4/Task2_2.js]:
```suggestion
if (typeof age !== 'number') {
```

If follow task description we only need to check type of value

File: HW_Lection4/Task2_3.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+// 3.
+// Преобразовать Task 2 - 1 таким образом, чтобы значение НАПРИМЕР '2' (т.е. ЛЮБАЯ строка в которой лежат ТОЛЬКО ЦИФРЫ) пропускалось, 
+//   преобразовываясь в number
+
+let age = '10';
+let age_2 = 18;
+let age_3 = 61;
+
+if (typeof age === 'number') {
```
Comment (VladimirLQA) [inline HW_Lection4/Task2_3.js]:
```suggestion
if (!isNaN(age)) {
```

Add output to `else` block for this case

File: HW_Lection4/Task2_3.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        case age >= age_2 && age < age_3:
+            console.log('Welcome !');
+            break;
+        case age > age_3:
+            console.log('Keep calm and look Culture channel');
+            break;
+        default:
+            console.log('Technical work');
+    }
+}
```
Comment (VladimirLQA) [inline HW_Lection4/Task2_3.js:25]:
```suggestion
} else { ... }
```

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

File: HW_Lection4/Task2_2.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
+// 2. 
+// Преобразовать написанный код в task 1 так, чтобы сначала проверялся тип данных. 
+//   И если он не number - кидалась ошибка в консоль.
+//   Проверить работу кода на следующих данных 17, 18, 61, "2", "aaa"
+
+const age = '10';
+const age_2 = 18;
+const age_3 = 60;
+
+if (!isNaN(age)) {
```
Comment (VladimirLQA) [inline HW_Lection4/Task2_2.js]: This check need for 3-rd task

File: HW_Lection4/Task2_3.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+// 3.
+// Преобразовать Task 2 - 1 таким образом, чтобы значение НАПРИМЕР '2' (т.е. ЛЮБАЯ строка в которой лежат ТОЛЬКО ЦИФРЫ) пропускалось, 
+//   преобразовываясь в number
+
+const age = `10`;
+const age_2 = 18;
+const age_3 = 61;
+
+if (typeof age === `number`) {
```
Comment (VladimirLQA) [inline HW_Lection4/Task2_3.js]: And this check for 2-nd :)

**Review by VladimirLQA — APPROVED** [review]

**Review by anatoly-karpovich — APPROVED** [review]

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**